pkg/commands/componentarchive/create_test.go

```diff
@@ -0,0 +1,58 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package componentarchive_test
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+	"github.com/mandelsoft/vfs/pkg/layerfs"
+	"github.com/mandelsoft/vfs/pkg/memoryfs"
+	"github.com/mandelsoft/vfs/pkg/osfs"
+	"github.com/mandelsoft/vfs/pkg/projectionfs"
+	"github.com/mandelsoft/vfs/pkg/vfs"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/gardener/component-cli/pkg/commands/componentarchive"
+	cdv2 "github.com/gardener/component-spec/bindings-go/apis/v2"
+	"github.com/gardener/component-spec/bindings-go/codec"
+)
+
+var _ = Describe("Create", func() {
+
+	var testdataFs vfs.FileSystem
+
+	BeforeEach(func() {
+		baseFs, err := projectionfs.New(osfs.New(), "./testdata")
+		Expect(err).ToNot(HaveOccurred())
+		testdataFs = layerfs.New(memoryfs.New(), baseFs)
+	})
+
+	It("should create a component archive", func() {
+		opts := &componentarchive.CreateOptions{}
+		opts.Name = "example.com/component/name"
+		opts.Version = "v0.0.1"
+		opts.ComponentArchivePath = "./create-test"
+
+		fileName := "component-descriptor.yaml"
```

Comment:
you should use the constant from the component-spec instead: https://github.com/gardener/component-spec/blob/master/bindings-go/ctf/ctf.go#L32

Reply:
ok, will be fixed